### ISSUE TITLE
GKJ abort

### ISSUE THREAD

**Issue by jmirabel** [humanoid-path-planner/hpp-fcl#48]
I encountered the following abort:
https://github.com/humanoid-path-planner/hpp-fcl/blob/7c65d1ef1a369b4b80d598cfd1a837f96d948efc/src/narrowphase/narrowphase.cpp#L678

The GJK inputs are:
- Triangle 1 at 
```
  a = Eigen::Matrix<double,3,1,ColMajor> (data ptr: 0x7ffff13e0f38) = {
    [0] = 0.063996093749999997,
    [1] = -0.15320971679687501,
    [2] = -0.42799999999999999
  }, 
  b = Eigen::Matrix<double,3,1,ColMajor> (data ptr: 0x7ffff13e0f50) = {
    [0] = 0.069105957031249998,
    [1] = -0.150722900390625,
    [2] = -0.42999999999999999
  }, 
  c = Eigen::Matrix<double,3,1,ColMajor> (data ptr: 0x7ffff13e0f68) = {
    [0] = 0.063996093749999997,
    [1] = -0.15320971679687501,
    [2] = -0.42999999999999999
  }
```
at
```
  T = Eigen::Matrix<double,3,1,ColMajor> (data ptr: 0x7ffff13e20e8) = {
    [0] = -12.824601270753471,
    [1] = -1.6840516940066426,
    [2] = 3.8914453043793844
  }, 
  q = Eigen::Quaternion<double> (data ptr: 0x7ffff13e2100) = {
    [x] = -0.26862477561450587,
    [y] = -0.46249645019513175,
    [z] = 0.73064726592483387,
    [w] = -0.42437287410898855
  }
```
- Triangle 2:
```
  a = Eigen::Matrix<double,3,1,ColMajor> (data ptr: 0x7ffff13e0ff8) = {
    [0] = -25.655000000000001,
    [1] = -1.2858199462890625,
    [2] = 3.7249809570312502
  }, 
  b = Eigen::Matrix<double,3,1,ColMajor> (data ptr: 0x7ffff13e1010) = {
    [0] = -10.926,
    [1] = -1.284259033203125,
    [2] = 3.7281499023437501
  }, 
  c = Eigen::Matrix<double,3,1,ColMajor> (data ptr: 0x7ffff13e1028) = {
    [0] = -10.926,
    [1] = -1.2866180419921875,
    [2] = 3.72335400390625
  }
```
at
```
  T = Eigen::Matrix<double,3,1,ColMajor> (data ptr: 0x7ffff13e2198) = {
    [0] = 0,
    [1] = 0,
    [2] = 0
  }, 
  q = Eigen::Quaternion<double> (data ptr: 0x7ffff13e21b0) = {
    [x] = 0,
    [y] = 0,
    [z] = 0,
    [w] = 1
  }
```

**Comment by florent-lamiraux** [comment]
I will investigate. In the meantime, you can simply remove the abort.

**Comment by florent-lamiraux** [comment]
I have not been able to reproduce the issue in fcl tests. Can you checkout commit 331410e2e4efa13573fa2fbe8f09f210be8ea680 in my space and double check test/test_fcl_gjk.cpp where I have introduced the case above.

Running test_fcl_gjk.py with gepetto-gui running should display the triangles that seem very close to each other.

**Comment by florent-lamiraux** [comment]
After double checking, I have reproduced the bug. @jmirabel : do not take into account of my previous comment.

**Comment by jmirabel** [comment]
I believe this has been fixed by baa374228647ff52c8be68be77cdf8bd46227b04.
